src/scss/components/_header.scss

```diff
@@ -0,0 +1,248 @@
+$z-index--1010: 1010;
+$z-index--1009: 1009;
+
+$letter-spacing--0-2: -0.2px;
+
+$line-height--1-2: 1.2;
+
+$color-border-default: #dcdacb;
+$color-border-dark: #989586;
+$color-border-blue:#2896a9;
+
+$max-width--122: 122px;
+$min-width--240:240px;
+
+$padding--0--12: 0 12px;
+$padding--0--16: 0 16px;
+$padding--8--16: 8px 16px;
+$padding-right--12: 12px;
+$padding-left--4: 4px;
+$padding-bottom--8: 8px;
+
+$margin--0--12: 0 12px;
+$margin-left--8: 8px;
+
+.header-top {
+    align-items: center;
+    display: flex;
+    background: $color-white;
+    height: $height--72;
+    padding: $padding--0--16;
+    position: relative;
+    z-index: $z-index--1010;
+
+    .logo {
+        align-items: center;
+        display: flex;
+        flex-shrink: 0;
+        padding-right: $padding-right--12;
+        max-width: $max-width--122;
+    }
+
+    .nav-category {
+        font-weight: $font-weight-nomal;
+
+        p {
+            padding: $padding--0--12;
+            margin: $margin--0--12;
+        }
+    }
+
+    .box-search {
+        flex-grow: 1;
+        height: $height--48;
+        border: $border-size-default solid $color-border-dark;
+        border-radius: $border-radius--9999;
+        background-color: $background-color-dark-gray;
+        margin: $margin--0--12;
+
+        .udlite-search-form {
+            display: flex;
+            align-items: center;
+            height: $height-full;
+            flex-direction: row-reverse;
+
+            .udlite-text-input {
+                border-radius: $border-radius--4;
+                border: $border-size-default solid $color-border-dark;
+                display: block;
+                padding: $padding--0--16;
+                min-width: $min-width--240;
+                width: $width-full;
+                background-color: transparent;
+                padding-left: $padding-left--4;
+                border: 0;
+                flex: 1;
+                min-width: 0;
+                height: $height--40;
+            }
+
+            .udlite-form-label {
```

Comment:
Should you rename this case to form-label or something, Please make all variable and classname is make sense